competencies/urls.py

```diff
@@ -0,0 +1,17 @@
+from django.urls import path
+
+from competencies.views import self_assess_competencies, list_competencies, \
```

Comment:
Wouldn't it just be better to say:

`from competencies import views`

Then below:
`path('', views.self_assess_competencies, name="self_assess_competencies"),`

Reply:
Changed as suggested.